Application/EdFi.Ods.Api/Models/AssemblyMetadata.cs

```diff
@@ -0,0 +1,12 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.Ods.Api.Models
+{
+    public class AssemblyMetadata
```

Comment:
The location of this implies that it is part of the API's models, when in fact it is just be used as a object for deserializing some custom assembly metadata for internal purposes during initialization. It really has no general purpose.

Please remove this file, but add the class as a `private class` at the bottom of _AssemblyLoaderHelper.cs_ file, like so:
```csharp
        ...

        private class PluginAssemblyLoadContext : AssemblyLoadContext
        {
            public PluginAssemblyLoadContext()
                : base(isCollectible: true) { }
        }
        
        private class AssemblyMetadata
        {
            public string AssemblyModelType { get; set; }
        }
    }
}
```

Reply:
@semalaiappan Looks like you missed this one. Please move the `AssemblyMetadata` class as described in this comment, and make it private.